curriculum/challenges/english/20-upcoming-python/learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm/656877f71bba2b97acedf9af.md

```diff
@@ -0,0 +1,65 @@
+---
+id: 656877f71bba2b97acedf9af
+title: Step 3
+challengeType: 20
+dashedName: step-3
+---
+
+# --description--
+
+Python comes with built-in classes that can help us with string manipulation. One of them is the `str` class. It has a method called `maketrans` that can help us create a translation table. This table can be used to replace characters in a string:
+
+```python
+str.maketrans({'t': 'c', 'l': 'b'})
+```
+
+The above, when called on a string, will replace all `t` characters with `c` and all `l` characters with `b`.
```

Comment:
I don't have wording suggestion for this, but instead of adding what later can be done with it (that's in the next step), maybe just focus on how the resulting dictionary looks like - _unicode ordinal of key character_ -> _character_

Reply:
Sorry @gikf , I am not smart enough to understand what this means:
> how the resulting dictionary looks like - unicode ordinal of key character -> character

😅

Reply:
:stuck_out_tongue: I should have just show the example
```
>>> str.maketrans({'t': 'c', 'l': 'b'})
{116: 'c', 108: 'b'}
```